### PR TITLE
Move cabinet tests to the appropriate modules

### PR DESCRIPTION
**Pull request by magiccookie**
#2408

### CODE REVIEW

File: apps/ehealth/test/web/controllers/cabinet/declaration_controller_test.exs [hunk 1]
```diff
@@ -5,12 +5,98 @@ defmodule EHealth.Web.Cabinet.DeclarationControllerTest do
 
   alias EHealth.MockServer
 
+  setup :verify_on_exit!
+
+  defmodule OpsServer do
```
Comment (AlexKovalevych) [inline apps/ehealth/test/web/controllers/cabinet/declaration_controller_test.exs]: please get rid of OpsServer, use Mox

Comment (magiccookie) [reply apps/ehealth/test/web/controllers/cabinet/declaration_controller_test.exs]: fixed

File: apps/ehealth/test/web/controllers/cabinet/declaration_request_controller_test.exs [hunk 2]
```diff
@@ -1,10 +1,12 @@
 defmodule EHealth.Web.Cabinet.DeclarationRequestControllerTest do
   @moduledoc false
 
-  use EHealth.Web.ConnCase
+  use EHealth.Web.ConnCase, async: false
```
Comment (AlexKovalevych) [inline apps/ehealth/test/web/controllers/cabinet/declaration_request_controller_test.exs]: why do you need it to be synchronous?

File: apps/ehealth/test/web/controllers/cabinet/declaration_request_controller_test.exs [hunk 3]
```diff
@@ -731,40 +927,48 @@ defmodule EHealth.Web.Cabinet.DeclarationRequestControllerTest do
...
-      "level" => Enum.random(doctor_levels()),
-      "qualification_type" => Enum.random(doctor_qualification_types()),
-      "attestation_name" => "random string",
-      "attestation_date" => ~D[1987-04-17],
-      "valid_to_date" => ~D[1987-04-17],
-      "certificate_number" => "random string"
-    }
-    |> Map.merge(params)
-  end
+  defp get_person2(id, response_status, params) do
```
Comment (AlexKovalevych) [inline apps/ehealth/test/web/controllers/cabinet/declaration_request_controller_test.exs]: Please use a single function to get person, get rid of `mpi_get_person` and `get_person2`

File: apps/ehealth/test/web/controllers/cabinet/declaration_request_controller_test.exs (hunk 2 above)
Comment (magiccookie) [reply apps/ehealth/test/web/controllers/cabinet/declaration_request_controller_test.exs]: fixed

File: apps/ehealth/test/web/controllers/cabinet/declaration_request_controller_test.exs (hunk 3 above)
Comment (magiccookie) [reply apps/ehealth/test/web/controllers/cabinet/declaration_request_controller_test.exs]: fixed

**Pull request merged**